bench/workgen/workgen.cxx

```diff
@@ -774,53 +775,56 @@ int ThreadRunner::op_run(Operation *op) {
     }
     // Retry on rollback until success.
     while (retry_op) {
-	if (op->_transaction != NULL) {
+        if (op->_transaction != NULL) {
             if (_in_transaction)
                 THROW("nested transactions not supported");
             _session->begin_transaction(_session,
```

Comment:
This isn't part of your changes, but the call to `begin_transaction` should be inside a `WT_ERR` macro.

Reply:
Fixed.